### PR TITLE
feature(assertions): Add 'isTrue' since it is truly needed

### PR DESCRIPTION
**Pull request by andreasmarkussen**
Make this code (boilerplate~ish) at little nicer and shorter, while also making the Text in the test report more to the point. See changes in the test spec. 

    .equals(true)

And just write

     .isTrue()

And the opposite has also been added 

    .isFalse()

### CODE REVIEW

File: packages/assertions/src/expectations/isTrue.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Answerable } from '@serenity-js/core';
+
+import { Expectation } from '../Expectation';
+
+export function isTrue(): Expectation<boolean> {
+  return Expectation.thatActualShould<boolean, boolean>(`have value that is`, true)
```
Comment (jan-molak) [inline packages/assertions/src/expectations/isTrue.ts:6]: I think if you used an alias instead we might get a nicer error message should the assertion fail:

```typescript
export function isTrue(): Expectation<boolean> {
    return Expectation.to(`be true`)
        .soThatActual(equals(true))
}
```

File: .gitignore [hunk 2]
```diff
@@ -23,3 +23,5 @@ packages/*/target/
 target/
 *.log
 packages/*/lib
+coverage
```
Comment (jan-molak) [inline .gitignore:26]: Hmm, is there any part of the build process that writes to `./coverage`? Or are those reports caused by an IDE you use? (I don't mind having that ignored, just curious 😄)

File: packages/assertions/src/expectations/index.ts [hunk 3]
```diff
@@ -6,6 +6,7 @@ export * from './endsWith';
 export * from './equals';
 export * from './includes';
 export * from './isAfter';
+export * from './isTrue';
```
Comment (jan-molak) [inline packages/assertions/src/expectations/index.ts:9]: Apologies for nitpicking, but could we maintain alphabetical order of exports, please?

File: .gitignore (hunk 2 above)
Comment (andreasmarkussen) [reply .gitignore:26]: Removed - I think it came from VSCode on windows.

File: packages/assertions/src/expectations/index.ts (hunk 3 above)
Comment (andreasmarkussen) [reply packages/assertions/src/expectations/index.ts:9]: Fixed

File: packages/assertions/src/expectations/isTrue.ts (hunk 1 above)
Comment (andreasmarkussen) [reply packages/assertions/src/expectations/isTrue.ts:6]: Hi @jan-molak, 
I would love that short text as well, 

The error message - `be true` is put in front of does..

>  2) isTrue
>       contributes to a human-readable description:
>
>      AssertionError: expected '#actor ensures that true does be true' to equal '#actor ensures that true does have value that is true'
>      + expected - actual
>
>      -#actor ensures that true does be true
>      +#actor ensures that true does have value that is true

And the "does" comes from Ensure.toString() and Check.toString()

So I need to do a bigger rewrite if, so "does" is not a part of the picture.

Comment (jan-molak) [reply packages/assertions/src/expectations/isTrue.ts:6]: True, good point. Sorry, I was doing that review in the middle of the night so missed that :-)

Comment (andreasmarkussen) [reply packages/assertions/src/expectations/isTrue.ts:6]: Just for later reference: 
The code I got working before I realized the larger changes where 

     export function isTrue() {
       return Expectation.to(`have value that is true`)
     .soThatActual(equals(true))
    }

**Comment by andreasmarkussen** [comment]
Added isFalse() - have been busy with other things, but here it is :)

**Comment by jan-molak** [comment]
Awesome, thanks for your contribution, @andreasmarkussen!

**Pull request closed without merging**